product_docs/docs/pgd/4/admin-manual/installing/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploying manually
+navigation:
+- 01-provisioning-hosts
+- 02-install-postgres
+- 03-configuring-repositories
+- 04-installing-software
+- 05-creating-cluster
+- 06-check-cluster
+- 07-using-pgd-cli
+---
+
+EDB offers automated PGD deployment using TPA (Trusted Postgres Architect) because its generally more reliable than manual processes. 
```

Comment:
```suggestion
EDB offers automated PGD deployment using TPA (Trusted Postgres Architect) because it's generally more reliable than manual processes. '
```